### DISCUSSION TITLE
posts/15839/

### DISCUSSION THREAD

**Discussion by giscus** [STEVE-1936550490/blog_comment#8] [Announcements]
# posts/15839/

うた（歌）ーーーーー歌曲
 カラオケ　ーーーーー卡拉OK
 ロック　ーーーーー摇滚乐
 ポップス　ーーーーー流行音乐
 クラシック　ーーーーー西方古典音乐
 ピアノ　ーーーーー钢琴
 え（絵）ーーーーー画
 えいご（英語）ーーーーー英语
 スペイン語　ーーーーー西班牙语
 すいえい（水泳）ーーーーー游泳
 スポーツ　ーーーーー体育
 ゴルフ　ーーーーー高尔夫
 うんてん（運転）ーーーーー开车
 の

https://steve-1936550490.github.io/posts/15839/

**Comment by STEVE-1936550490** [comment]
1
